src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -16,22 +20,22 @@ protected void countIn(int in) {
     }
 
     protected boolean callCheck() {
-        return total == -1;
+        return total == TOTAL_BEGINNING_NUMBER;
     }
 
     protected boolean isFull() {
-        return total == 11;
+        return total == (NUMBERS_AMOUNT - 1);
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return TOTAL_BEGINNING_NUMBER;
```

Comment:
Stałe o wartości -1 odnoszące się do pola total (też w metodzie callCheck), to nie jest ta sama stała o wartości -1 oznaczające domyślną zwracaną przez wyliczankę wartość (w metodach peekaboo i countOut). Powinny to być osobne stałe.